### PR TITLE
Updates/fixes to NTSL

### PR DESCRIPTION
**Pull request by TedStudley**
This pull request fixes the broken `/` NTSL division operator and adds several new math functions to NTSL, namely `sin`, `cos`, `asin`, `acos`, and `log`.

### CODE REVIEW

File: code/modules/scripting/Scanner/Scanner.dm [hunk 1]
```diff
@@ -118,13 +118,14 @@
 		for(, src.codepos<=lentext(code), src.codepos++)
 
 			var/char=copytext(code, codepos, codepos+1)
+			var/nextchar=copytext(code, codepos+1, codepos+2)
```
Comment (optimumtact) [inline code/modules/scripting/Scanner/Scanner.dm]: What if the / is the last char will this code still work?

Comment (TedStudley) [inline code/modules/scripting/Scanner/Scanner.dm]: If it's the last character on the string it generates a compile warning in the NTSL IDE:
```
>28: Unexpected token: /
```
It seems to not have any issues there.

**Comment by optimumtact** [comment]
:+1: good work

**Comment by Miauw62** [comment]
There's still a bug with assigning $message when $message is in italics, could you fix this (or do you have any idea where the code is located so i can fix it?)

**Comment by TedStudley** [comment]
I've figured out the problem, but I'll move discussion of that to the issue for it (#5833). I'll push the fix here once I've got that.

**Pull request merged**